components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -14928,28 +14928,30 @@ public void addAPIRegistryEntryMappings(int apiId, Map endpointRegistryEntries,
     }
 
     /**
-     * @param apiId                   ID of the API created using the Endpoint Registry Entry
-     * @param endpointRegistryEntries Map contains production and sandbox endpoint ID
-     * @throws APIManagementException
+     * @param apiId                   ID of the API created using the Service Catalog Entry
+     * @param endpointRegistryEntries HashSet contains production and sandbox endpoint IDs
```

Comment:
Update Javadocs, The set contains failover and sandbox endpoint Ids, right?

Reply:
Fixed in https://github.com/wso2/carbon-apimgt/pull/8728/commits/216e08f31573e70a482b6f9e3235b139d06c07bf

Reply:
Set can contain production/sandbox IDs of Load-balanced/Failover/Normal endpoints